File: Iniciante/1985.java
package Iniciante;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);
        int produtos = ler.nextInt();
        int codigo;
        int contador = 0;
        int quantidade;
        double preco = 0;
        double p1=0;
        while (contador < produtos) {
            codigo = ler.nextInt();
            quantidade = ler.nextInt();
            if (codigo==1001) {
                p1=quantidade*1.5;
            }
            else if (codigo==1002) {
                p1 = quantidade*2.5;
            }
            else if (codigo==1003) {
                p1=quantidade*3.5;
            }
            else if (codigo==1004) {
                p1=quantidade*4.5;
            }
            else if (codigo==1005) {
                p1=quantidade*5.5;
            }
            contador++;
            preco+=p1;
        }
        System.out.printf("%.2f\n",preco);
    }

}

File: Iniciante/1985.py
produtos = int(input())
contador = 0
preco_total = 0

while contador < produtos:
    codigo, quantidade = map(int, input().split())
    preco_produto = 0

    if codigo == 1001:
        preco_produto = quantidade * 1.5
    elif codigo == 1002:
        preco_produto = quantidade * 2.5
    elif codigo == 1003:
        preco_produto = quantidade * 3.5
    elif codigo == 1004:
        preco_produto = quantidade * 4.5
    elif codigo == 1005:
        preco_produto = quantidade * 5.5

    contador += 1
    preco_total += preco_produto

print('{:.2f}'.format(preco_total))
